_pages/cfp.md

```diff
@@ -5,7 +5,8 @@ subtitle: Call for Proposals
 permalink: /cfp/
 ---
 
-Our CFP will open on August 14 2017 and close on September 15 2017! 
+Our call for proposals (CFP) for 15 minute lightning talks will open on August 14 2017 and close on September 15 2017!
```

Comment:
hmm i thought the 1 month during which the cfp will be open would be enough but this makes sense too!